.github/workflows/actionlint.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   actionlint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3
```

Comment:
v3 is not updated…

Reply:
Please ignore renovate PRs for now

Reply:
Strange, it worked for stale: https://github.com/nextcloud/android-config/pull/28

Reply:
Ah, I see. It didn't update here because it's a [minor update](https://github.com/actions/checkout/releases) 3.1.0 -> 3.3.0), so it's still v3.

Reply:
Either way, this will become a non-issue once we use full tags: https://github.com/nextcloud/android-config/pull/27